docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/breadcrumb/breadcrumb.twig

```diff
@@ -20,11 +21,12 @@
   {% set links_count = links|length %}
   {% set parent = false %}
 
-  <nav class="civictheme-breadcrumb {{ modifier_class }}" aria-label="breadcrumb">
+  <nav class="civictheme-breadcrumb {{ modifier_class }}" {{ attributes }} aria-label="breadcrumb">
```

Comment:
@nickgeorgiou 
Can you please refactor this to use horizontal `item-list` to output items

Sorry - it was not in SD but need to be implemented this way